pkg/adapter/sigproxy_linux.go

```diff
@@ -30,6 +30,7 @@ func ProxySignals(ctr *libpod.Container) {
 				if err := syscall.Kill(syscall.Getpid(), s.(syscall.Signal)); err != nil {
```

Comment:
Wait, what is this even doing? Why are we double-sending the signal to ourself on errors?

Reply:
From git-blame, it looks like an attempt to make C-c kill Podman if sig-proxy fails because the container is stopped. I don't know if this is a good idea, or necessary at all, but it's probably not harmful

Reply:
I had the very same question and found https://github.com/containers/libpod/pull/1086/files#r202198031.

Reply:
But I am not sure about the correctness after all

Reply:
Should this be the container?

Reply:
Yes, that should be the container.

Reply:
This makes (some) sense to me as is — as long as the container is running, attempts to gracefully terminate Podman are redirected to gracefully terminating the container. (The downside is that if the container is uncooperative, it’s difficult to terminate Podman.)

If the container dies, and we find out here, we need to forward that one signal to ourselves so that it is not lost, and then we terminate the proxy and let the defaults play out.

(Pedantically, at the time of `StopCatch`, several other signals may be queued in `sigBuffer`, and we throw them away. For the typical `SIGINT`/`SIGTERM` case this doesn’t matter much.)

Reply:
Tests are green now. Mind dropping a LGTM if you agree to the changes?